src/daemons.ts

```diff
@@ -0,0 +1,86 @@
+import logger from './utilities/logger';
+import { getHeapStatistics } from 'v8';
+import configurations from './configurations';
+const {
+    debugThreshold,
+    warningThreshold,
+    errorThreshold,
+    interval: memoryDaemonInterval
+} = configurations.monitoring.memory;
+const minThreshold = Math.min(debugThreshold, warningThreshold, errorThreshold);
```

Comment:
Is the expected way to avoid log output to set all the environment variables to 100?

Reply:
Yup